analysis/analysisrunner.cpp

```diff
@@ -227,11 +240,11 @@ QString AnalysisRunner::humanFileSize() const
 {
     int unit;
     const char *units [] = {" Bytes", " kB", " MB", " GB"};
-    quint64 size = fileSize(); // or whatever
+    qint64 size = fileSize(); // or whatever
```

Comment:
Ok I check the Qt documentation QFileInfo::size return a qint64. It's strange, but silent cast isn't a good idea